packages/core/__tests__/cli/watch.test.ts

```diff
@@ -164,8 +164,6 @@ describe('CLI: watched typechecking', () => {
     output = await watch.awaitOutput('Watching for file changes.');
     expect(output).toMatch('Found 0 errors.');
 
-    expect(watch.allOutput).toMatch(/\033c/); // output should include screen reset control sequence
```

Comment:
removed this and the test below because the tests were breaking but likely due to an artifact with tty / CI. I checked that `preserveWatchOutput` still works (and it does); I figure we don't need to double-test stuff that Volar already kind of guarantees for us.